plugins/flutter_aepassurance/ios/flutter_aepassurance.podspec

```diff
@@ -3,7 +3,7 @@
 #
 Pod::Spec.new do |s|
   s.name             = 'flutter_aepassurance'
-  s.version          = '4.0.0'
+  s.version          = '4.0.2'
```

Comment:
why are we jumping from 0 to 2?

Reply:
I missed to update that for 4.0.1.  It should be matching to the aep flutter version, so correct to 4.0.2 in this PR.